docs/cis_self_assessment.md

```diff
@@ -749,23 +764,20 @@ When you create a pod, if you do not specify a service account, it is automatica
 
 **Result:** Pass
 
-**Remediation:**
-Follow the documentation and create ServiceAccount objects as per your environment.
-Then, edit the API server pod specification file `/etc/kubernetes/manifests/kube-apiserver.yaml`
-on the master node and ensure that the `--disable-admission-plugins` parameter is set to a
-value that does not include `ServiceAccount`.
-
 **Audit:**
+Run the below command on the master node.
 
-```
+```bash
 /bin/ps -ef | grep kube-apiserver | grep -v grep
 ```
 
-**Expected result**:
+Verify that the `--disable-admission-plugins` argument is set to a value that does not includes `ServiceAccount`.
 
-```
-'NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,DefaultTolerationSeconds,MutatingAdmissionWebhook,ValidatingAdmissionWebhook,ResourceQuota,NodeRestriction,Priority,TaintNodesByCondition,PersistentVolumeClaimResize,PodSecurityPolicy,EventRateLimit' has 'ServiceAccount' OR '--enable-admission-plugins' is not present
-```
+
+**Remediation:**
+By default, RKE2 does not use this argument. If there's a desire to use this argument, follow the documentation and create ServiceAccount objects as per your environment. Then, edit the API server pod specification file `/var/lib/rancher/rke2/agent/pod-manifests/kube-apiserver.yaml`
```

Comment:
Oh, this one is a little bit wrong because with RKE2 users cannot directly edit the pod manifest file. RKE2 will stomp on it on restart. They have to set it via the `kube-apiserver-arg` flag.  You can make it more generic like "Then refer to RKE2's documentation to see how to supply additional api server configuration via the kube-apiserver-arg parameter."

Reply:
you could just simplify down to "no manual blah blah blah"